src/raftstore/store/peer.rs

```diff
@@ -589,6 +595,31 @@ impl Peer {
         pending_peers
     }
 
+    pub fn peer_catch_up(&mut self, peer_id: u64) -> bool {
+        if self.peer_pending_after.is_empty() {
+            return false;
```

Comment:
Why false?

Reply:
The method means `if any new peers catch up, return true, otherwise return false`. Maybe rename it as `any_new_peer_catch_up` is better?

Reply:
If a store restarts after fail, there wil lbe too many snapshots if we return `true` here?